plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -231,3 +608,1407 @@ def set_wins_registration(self, mgrattr):
         if not response['ret']:
             return response
         return {'ret': True, 'changed': True, 'msg': "Modified %s" % mgrattr['mgr_attr_name']}
+
+    def verify_drive_count(self, raid_details, logical_drives_count):
+        if len(raid_details) != logical_drives_count:
+            return {
+                "ret": False,
+                "changed": False,
+                "msg": "Logical drive count in raid details is not matching with logical drives present in the server"
+            }
+        return {
+            "ret": True,
+            "msg": "Drive count is same as input"
+        }
+
+    def verify_logical_drives(self, raid_details, check_length=False):
+        # This method verifies logical drives present in the OOB controller against the provided input
+        result = self.get_logical_drives_details()
```

Comment:
I think this needs to be updated to use the new method name.

Reply:
Yes, updated. Thanks!

Reply:
Resolving this conversation. Thanks!